tests/e2e/scripts/hermes_bootstrap.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -ex
+
+# initialize Hermes relayer configuration
+mkdir -p /root/.hermes/
+touch /root/.hermes/config.toml
+
+# setup Hermes relayer configuration
+tee /root/.hermes/config.toml <<EOF
```

Comment:
any thoughts of making copying this as a file from our repository? like a `test/hermies_config.toml` for example?